src/groupWith.js

```diff
@@ -0,0 +1,33 @@
+var _curry2 = require('./internal/_curry2');
+
+/**
+ * Takes a list and returns a list of lists where each sublist contains only equal elements.
+ *
+ * @func
+ * @memberOf R
+ * @category List
+ * @sig (a, a -> Boolean) -> [a] -> [[a]]
+ * @param {Function} fn Function :: a, a -> Boolean
```

Comment:
I don't think we should include Hindley–Milner notation in `@param` descriptions. Users who rely on these descriptions likely haven't yet learnt to read HM.